From 7784929b5f146d4685c72dc3d71626adf1b6ac96 Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@z3ntu.xyz>
Date: Sun, 12 Feb 2023 12:41:38 +0100
Subject: [PATCH] Set default C++ version to C++17

Newest googletest requires at least C++14, but let's update to C++17
already since that's also required by Qt6.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c30aaa5..6fc145e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -45,7 +45,7 @@ endif()
 set(LOCALE_DIR "${CMAKE_INSTALL_FULL_DATADIR}/locale")
 
 add_definitions(
-	-std=c++11
+	-std=c++17
 	-fno-permissive
 	-pedantic
 	-Wall
-- 
2.42.0

